requirements_test_all.txt

```diff
@@ -3,19 +3,19 @@
 # make new things fail. Manually update these pins when pulling in a
 # new version
 asynctest==0.13.0
-black==19.3b0
-codecov==2.0.15
-coveralls==1.2.0
-flake8-docstrings==1.3.1
-flake8==3.7.8
-mock-open==1.3.1
-mypy==0.720
-pre-commit==1.18.3
-pydocstyle==4.0.1
-pylint==2.3.1
+black==20.8b1
```

Comment:
I don't think black, I sort, flake8 dependencies could be omitted here, as pre-commit would take care to install the correct one. And allows you to keep those up-to-date in a single place -- pre-commit config